Make aliases the same for SID and bigrams

demonoidv opened the pull request:
This is to be more consistent since those aliases will be used to fill the same field.
I think it's more convenient to have the exact same name for both `KnownAliasesDict` and `KnownSidsDict`.

demonoidv committed:
Make aliases the same for SID and bigrams

This is to be more consistent since those aliases will be used to fill
the same field.
I think it's more convenient to have the exact same name for both
`KnownAliasesDict` and `KnownSidsDict`.

zacateras commented:
Indeed there is mismatch between these two conventions and uppercased names contain less meaning. Since these two lists are referring to each other I suggest rewriting them to the single list of tuples like `(2-letter code, SID, meaningful name / description)`.

Please also ensure that unit tests are passing and README.md is up to date.

demonoidv committed:
Rewrite and rename `Match.OneByRegex()`

This function is now called `Match.OneByRegexOrPrefix()` and has been
rewrite to use a `List<(string, string, string)>` instead of the
original `Dictionary<string, string>`.
The goal of that is to prepare this fonction (called only to find a well
known alias for the `Sid` class) to handle a list of tuple wich contains
both bigram (when applicable) and the regex SID associated with the
alias name.

demonoidv committed:
Merge the two Dictionaries into single tuple List

demonoidv committed:
Correct tests to stay consistent with Sid mod

demonoidv committed:
Update README.md
